include/picongpu/plugins/IntensityPlugin.hpp

```diff
@@ -89,24 +89,24 @@ struct KernelIntensity
 
         if (threadId.x() == 0)
         {
-            /*clear destination arrays*/
+            // clear destination arrays
             s_integrated[threadId.y()] = float_X(0.0);
             s_max[threadId.y()] = float_X(0.0);
         }
         __syncthreads();
 
-        /*move cell wise over z direction(without garding cells)*/
+        // move cell-wise over z direction(without garding cells)
         for (int z = GuardSize::z::value * SuperCellSize::z::value; z < cellsCount.z() - GuardSize::z::value * SuperCellSize::z::value; ++z)
         {
-            /*move supercell wise over x direction without guarding*/
+            // move supercell-wise over x direction without guarding
             for (int x = GuardSize::x::value * SuperCellSize::x::value + threadId.x(); x < cellsCount.x() - GuardSize::x::value * SuperCellSize::x::value; x += SuperCellSize::x::value)
             {
                 const float3_X field_at_point(field(DataSpace<DIM3 > (x, yGlobal, z)));
                 s_field(threadId) = math::abs2(field_at_point);
                 __syncthreads();
                 if (threadId.x() == 0)
                 {
-                    /*master threads moves cell wise over 2D supercell*/
+                    // master threads moves cell-wise over 2D supercell
```

Comment:
`threads moves` is not correct. As this probably refers to a single thread per block i'm actually not sure if single or plural is more approptiate here.

Reply:
Yes it must be `master thread` it is only thread zero